Packs/OpsGenie/Integrations/OpsGenieV3/OpsGenieV3.yml

```diff
@@ -1500,6 +1516,31 @@ script:
     - contextPath: OpsGenie.Team.name
       description: Team name.
       type: String
+  - name: opsgenie-get-request
+    polling: true
+    description: Get a request in Opsgenie
+    deprecated: false
+    arguments:
+      - name: output_prefix
+        deprecated: true
+        description: The output prefix to use for the results
+      - name: polled_once
+        deprecated: true
+        description: 'Whether this is the first time polling'
+        required: false
+      - name: request_id
+        description: The id of the request to get
+        required: true
+        isArray: false
+      - name: request_type
+        description: The type of the request to get
+        required: true
+        isArray: false
+        auto: PREDEFINED
+        predefined:
+          - alerts
+          - incedents
+    outputs: []
```

Comment:
we do have have outputs, no?

Reply:
Outputs depend on the input. Should both be listed here?

Reply:
lets discuss

Reply:
yes